src/leiningen/new/tenzing.clj

```diff
@@ -167,9 +171,14 @@
 (defn warn-on-exclusive-opts!
   "Some options can't be used together w/o added complexity."
   [opts]
-  (when (and (om? opts)
+  (when (and (or (om? opts)
+                 (om-next? opts))
              (reagent? opts))
-    (main/warn "Please specify only +om or +reagent, not both.")
+    (main/warn "Please specify only +om/om-next or +reagent, not both.")
+    (main/exit))
+  (when (and (om? opts)
+             (om-next? opts))
+    (main/warn "Please specify only +om or +om-next, not both.")
```

Comment:
I guess we should generalize this to something that just checks that only one option has been selected.